doc/source/api/core/solver/settings.rst

```diff
@@ -1,77 +1,76 @@
 .. _ref_settings:
 
-Settings objects
-================
-Settings objects provide a natural way to access and modify Fluent settings and
-issue commands with a hierarchy of objects.
+Solver settings objects
+=======================
+Solver settings objects provide a natural way to access and modify Fluent solver
+settings and issue commands to be executed in the Fluent solver.
 
-Top-level settings objects
---------------------------
-The top-level settings object is available as the :ref:`root<settings_root_section>`
-property of ``session.solver``.
+Accessing solver settings
+-------------------------
+An appropriate call to ``launch_fluent`` returns an object (named ``solver`` in
+the following code snippets) whose interface directly exposes the
+:ref:`root<settings_root_section>` of the solver settings hierarchy.
 
 .. code-block::
 
   >>> import ansys.fluent.core as pyfluent
   >>> solver = pyfluent.launch_fluent(mode="solver")
-  >>> root = solver
 
 
-The ``root`` object contains attributes such as ``file``, ``setup``,
-``solution``, and ``results``.  These objects are also instances of settings
-objects and roughly mirror the outline view in Fluent.
+The ``solver`` object contains attributes such as ``file``, ``setup``, ``solution`` and
```

Comment:
```suggestion
The ``solver`` object contains attributes such as ``file``, ``setup``, ``solution``, and
```

Reply:
So, we are supposed to use the Oxford comma? Is that the standard? It's not considered standard English here.

Reply:
Yes--its standard in tradition Ansys doc and in the Google Developeper's Style Code. The Vale check that is implemented checks that the Oxford comma is there and raises an error if it isn't. The goal is to avoid any possible confusion. The book [Eats, Shoots and Leaves](https://www.amazon.com/Eats-Shoots-Leaves-Commas-Difference/dp/0399244913) refers to a famously misplaced comma in a wildlife manual that ended up suggesting a panda rather violently “eats, shoots and leaves” instead of eating shoots and leaves. Virtually all style guides now recommend using Oxford commas.